Gemfile.lock

```diff
@@ -319,4 +320,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.14.4
+   1.14.6
```

Comment:
Does adjusting the bundle version have any side-effects with Circle or Appraisal? This is something I've seen before (you might want to look at the run commands).